### ISSUE TITLE
Module: Version Attribute Missing?

### ISSUE THREAD

**Issue by ax3l** [openPMD/openPMD-viewer#128]
Hi @RemiLehe,

Does the main package of the viewer lack a `__version__` attribute, or am I missing it?

```python
import opmd_viewer

print(opmd_viewer.__version__)
# AttributeError: 'module' object has no attribute '__version__'
```

We might want to add it, there is are two PEP about it :)
https://github.com/ornladios/ADIOS/issues/61#issue-134347515

**Comment by RemiLehe** [comment]
The version is indicated in the `setup.py`, but you are correct that the module itself does not have a `__version__` attribute. However, we could definitely do it in `opmd_viewer/__init__.py`. I'll try to do a corresponding pull request.

**Comment by ax3l** [comment]
closed via #129, thank you! :)
